openedx/adg/lms/courseware_override/views.py

```diff
@@ -0,0 +1,77 @@
+"""
+Override courseware views file
+"""
+import pycountry
+
+from common.djangoapps.student.models import CourseEnrollment
+from common.djangoapps.student.roles import CourseInstructorRole
+from openedx.adg.lms.applications.models import MultilingualCourseGroup
+from openedx.core.djangoapps.user_api.accounts.image_helpers import get_profile_image_urls_for_user
+
+
+def get_course_enrollment_count(course_key):
+    """
+    Get all number of total enrollments in a course, given the course key.
+
+    Arguments:
+        course_key: The course key for which to fetch the enrollment count
+
+    Returns:
+        int: Number of total enrollments
+    """
+    enrollments = CourseEnrollment.objects.enrollment_counts(course_key)
+    return enrollments['total']
+
+
+def get_all_languages_for_course(course):
+    """
+    Get a list of all the languages a particular course exists in.
+
+    Arguments:
+        course (CourseOverview): A course object
+
+    Returns:
+        list: A list containing the language names as str
+    """
+    languages = []
+
+    course_language = pycountry.languages.get(alpha_2=course.language).name
+    languages.append(course_language)
+
+    multilingual_course_group = MultilingualCourseGroup.objects.filter(
+        multilingual_courses__course=course).first()
+
+    if multilingual_course_group:
+        for multilingual_course in multilingual_course_group.open_multilingual_courses:
+            course_language_code = multilingual_course.course.language
+            language_name = pycountry.languages.get(alpha_2=course_language_code).name
+            if language_name not in languages:
+                languages.append(language_name)
```

Comment:
I think the additional check inside the loop can be avoided by reorganising these statements. Correct me if I'm wrong.
```
    multilingual_course_group = MultilingualCourseGroup.objects.filter(
        multilingual_courses__course=course).first()

    if multilingual_course_group:
        for multilingual_course in multilingual_course_group.open_multilingual_courses:
            course_language_code = multilingual_course.course.language
            language_name = pycountry.languages.get(alpha_2=course_language_code).name
    else:
        course_language = pycountry.languages.get(alpha_2=course.language).name
        languages.append(course_language)
```

Reply:
I actually want the current course language to be at the start of the list.

I went with your approach too before.